### PR TITLE
ADD: Multi API Support

### PR DESCRIPTION
**Pull request by sihrc**

### CODE REVIEW

File: IndicoIo/Utils.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    }
+
+    public static function convertResults($results, $apis) {
+        $converted_results = array();
+        foreach ($apis as $api) {
+            $response = $results[self::$MAP_APIS[$api]];
+            if (array_key_exists("results", $response)) {
+                $converted_results[$api] = $response["results"];
+            } else {
+                $converted_results[$api] = $response["error"];
```
Comment (madisonmay) [inline IndicoIo/Utils.php]: I think it's important we keep the error key here?  I think that's how we handle python currently.

Comment (sihrc) [inline IndicoIo/Utils.php]: Well.. in the python, i think we just raise an exception :p

Comment (madisonmay) [inline IndicoIo/Utils.php]: Ah.  Yes that would make total sense.

Comment (sihrc) [inline IndicoIo/Utils.php]: Do the same?

Comment (madisonmay) [inline IndicoIo/Utils.php]: :+1:

**Comment by madisonmay** [comment]
Looks great to me.

**Pull request merged**